src/sentry/static/sentry/app/views/settings/organizationRepositories/addRepositoryLink.jsx

```diff
@@ -128,7 +128,7 @@ class AddRepositoryLink extends PluginComponentBase {
     const provider = this.props.provider;
     let hasIntegration = true;
     provider.config.forEach(field => {
-      if (field.initial == '') {
```

Comment:
This might be worth testing too?

Reply:
I changed it to a falsy check since it seems to be done like that in https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/settings/organizationRepositories/addRepositoryLink.jsx#L131

Reply:
@MeredithAnya Is this ok?